discovery/src/main/java/com/ibm/watson/discovery/v2/model/QueryResultMetadata.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * (C) Copyright IBM Corp. 2019.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.discovery.v2.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.cloud.sdk.core.service.model.GenericModel;
+
+/**
+ * Metadata of a query result.
+ */
+public class QueryResultMetadata extends GenericModel {
+
+  /**
+   * The document retrieval source that produced this search result.
+   */
+  public interface DocumentRetrievalSource {
+    /** search. */
+    String SEARCH = "search";
+    /** curation. */
+    String CURATION = "curation";
+  }
+
+  @SerializedName("document_retrieval_source")
+  private String documentRetrievalSource;
+  @SerializedName("collection_id")
+  private String collectionId;
+  private Double confidence;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/watson-developer-cloud/java-sdk/pullRequest?prid=4552601)